WordPress/gutenberg
# Replace 'should_skip_block_supports_serialization' with the core utility method

## What?
PR replace `gutenberg_should_skip_block_supports_serialization` with `wp_should_skip_block_supports_serialization`.

## Why?
The plugin now requires a minimum WP 6.0, so we can remove the `gutenberg_should_skip_block_supports_serialization` compat layer.

## Testing Instructions
1. Downgrade WP installation to WP 6.0 - `wp core update --version=6.0.3 --force`
2. Confirm there are no fatal errors when visiting the testing site - frontend and the editor.
